lua/telescope/_extensions/neorg/insert_link.lua

```diff
@@ -135,14 +135,14 @@ return function(opts)
 
                 vim.api.nvim_put(
                     {
-                        "["
-                            .. entry.ordinal:gsub(":$", "")
-                            .. "]"
-                            .. "("
+                        "{"
                             .. inserted_file
                             .. entry.display:gsub("^(%W+)%s+.+", "%1")
                             .. entry.ordinal:gsub("[%*#%|_]", "\\%1")
```

Comment:
Don't these need some updating, too?

Reply:
that's what I meant with
> atm there is no space after * etc
not sure where do insert that though (I'm bad a regex)
I could try